Header header
int32 width
int32 height
int32 scanRow
bool isLeftDetected
bool isRightDetected
int32 lpos
int32 rpos